apple/LiveMarkdownModule.mm

```diff
@@ -28,51 +24,17 @@ @implementation LiveMarkdownModule {
   RCTCxxBridge *cxxBridge = (RCTCxxBridge *)self.bridge;
   jsi::Runtime &rt = *(jsi::Runtime *)cxxBridge.runtime;
   expensify::livemarkdown::injectJSIBindings(rt);
-
-  RCTScheduler *scheduler = [surfacePresenter_ scheduler];
-  commitHook_ = std::make_shared<MarkdownCommitHook>(scheduler.uiManager);
-
+  
```

Comment:
No need for newlines
```suggestion
```

Reply:
[8a0dec9](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/8a0dec9a418c05faf46ad0d350047ee4c289ae96)